lib/installation_user_settings.pm

```diff
@@ -39,7 +39,13 @@ sub enter_userinfo {
     my (%args) = @_;
     $args{username} //= $realname;
     send_key 'alt-f';    # Select full name text field
-    wait_screen_change { $args{retry} ? type_string_slow $args{username} : type_string $args{username} };
+    wait_screen_change {
+        if (defined $args{max_interval}) {
+            type_string($args{username}, max_interval => $args{max_interval});
+        } else {
+            type_string($args{username});
+        }
```

Comment:
I think you can do something like this
```suggestion
$args{max_interval} //= undef;
...
    wait_screen_change {
            type_string($args{username}, max_interval => $args{max_interval});
```
Since the definition of type_string in [testapi sets it to 250](https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1408) if it's not defined

Reply:
I wanted to be explicit. I prefer to not send undef. If there is no parameter to send, better send no parameter at all.
But, if it is an issue for you I can change it.

Reply:
Matter of taste :)

Reply:
+1 to the @foursixnine suggestion